### PR TITLE
Rename package from talisman to flask_talisman

### LINKED ISSUE #3: How about renaming the package to flask-talisman?
For various reasons (conventions, OCD and more):

-------------------------------------

Not sure where I read it, but I think there is a convention for Flask extensions to name them with `flask` prefix.

For example one of my `requirements.txt` looks like this (which is pretty cool, because I know exactly which one has a dependency on Flask)

```
...
flask==0.11.1
flask-babel==0.11.1
flask-login==0.4.0
flask-oauthlib==0.9.3
flask-restful==0.3.5
flask-wtf==0.13.1
...
talisman==0.1.0
...
```

and directories look like this:

screen shot 2016-12-02 at 21 09 06

----------------------------

Even though there is no real conflict, there might be some confusion from the example in `README`.

screen shot 2016-12-02 at 21 21 06

----------------------------

Another reason is to avoid conflicts when importing modules instead of classes when we will have to import the `talisman` object to add extra rules to the endpoints. Because there is no better name to name this object, other than `talisman`.

If the coding convention is to `import module` (instead of `from module import something`) then the suggested example will have to be adjusted like this ([I agree with Martijn Pieters](http://softwareengineering.stackexchange.com/questions/187403/import-module-vs-from-module-import-function/187471)):

```python
from flask import Flask
import talisman as flask_talisman

app = Flask(__name__)
# Just to avoid having a different name of that variable for alter usages.
talisman = flask_talisman.Talisman(app) 

@app.route('/normal')
def normal():
    return 'Normal'

@app.route('/embeddable')
@talisman(frame_options=ALLOW_FROM, frame_options_allow_from='*')
def embeddable():
    return 'Embeddable'
```

---------------------------------

I understand that this will break a bunch of things, but in a long run this will be better, plus the actual name of this repo is already `flask-talisman`, so we should be able to install it like the rest of the Flask extensions: `pip install flask-talisman`.

### PR DESCRIPTION
**Pull request by lipis**
- Fixes #3 
- I never released a package before.. so please verify which changes had to be `flask_talisman` and which ones `flask-talisman`
- Updated the version to `1.0.0`
- Updated the URLs to `flask-talisman` in PyPi

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,9 +21,9 @@
 
 
 setup(
-    name='talisman',
+    name='flask-talisman',
 
-    version='0.1.0',
+    version='1.0.0',
```
Comment (lipis) [inline setup.py]: Let me know which one should be..

Comment (jonparrott) [inline setup.py]: Set it to 0.2.0 for now.

File: tox.ini [hunk 2]
```diff
@@ -9,7 +9,7 @@ deps =
 commands =
   nosetests \
     --with-coverage \
-    --cover-package talisman \
+    --cover-package flask-talisman \
```
Comment (jonparrott) [inline tox.ini]: This one should be an `_`.

**Comment by lipis** [comment]
there you go..

**Comment by jonparrott** [comment]
Can you if you can get travis to pass?

**Comment by lipis** [comment]
I tried a few things.. but I'm not familiar with the packages :( at least one of the 3 tests are passing now :)

**Comment by jonparrott** [comment]
I'll see if I can get it fixed tomorrow. :)

**Comment by jonparrott** [comment]
@lipis thanks for doing this!

**Comment by lipis** [comment]
Woohoo! Thank you.. :) Looks much better..
screen shot 2016-12-08 at 21 35 23

**Pull request merged**